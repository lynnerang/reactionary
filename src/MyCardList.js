import React from 'react';
import './MyCardList.scss';
import Card from './Card.js';

function MyCardList(props) {
    return (
      <>
        {
          props.myCards && props.myCards.map(card => {
            return <Card key={card.id}
                         hasTerm={true} 
                         canDelete={true} 
                         cardData={card}
                         deleteMyCard={props.deleteMyCard}
                   />
          })
        }
      </>
    )
}

export default MyCardList;