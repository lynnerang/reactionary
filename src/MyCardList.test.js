import React from 'react';
import MyCardList from './MyCardList.js';
import { shallow } from 'enzyme';


describe('MyCardList', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow (
      <MyCardList 
      />
      )
  });

  it('Should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});