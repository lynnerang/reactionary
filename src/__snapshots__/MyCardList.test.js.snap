// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyCardList Should match the snapshot 1`] = `<Fragment />`;
